pkgs/development/python-modules/xml2rfc/default.nix

```diff
@@ -1,11 +1,11 @@
-{ lib, fetchPypi, buildPythonPackage, intervaltree, pyflakes, requests, lxml, google-i18n-address
-, pycountry, html5lib, six, kitchen, pypdf2, dict2xml, weasyprint, pyyaml, jinja2
-, stdenv
+{ stdenv, fetchPypi, buildPythonPackage, pythonAtLeast, intervaltree, pyflakes, requests, lxml, google-i18n-address
+, pycountry, html5lib, six, kitchen, pypdf2, dict2xml, weasyprint, pyyaml, jinja2, ConfigArgParse, appdirs
 }:
 
 buildPythonPackage rec {
   pname = "xml2rfc";
   version = "3.5.0";
+  disabled = pythonAtLeast "3.9";
```

Comment:
@SuperSandro2000 Do you remember why you set this?

Reply:
Can't remember.